### DISCUSSION TITLE
Error on Stat: 2

### DISCUSSION THREAD

**Discussion by ankushnarula** [coqui-ai/STT#2208] [Q&A]
The documentation isn't very clear about M1 support for native client.  But in any case, I tried the following 

`% ./stt --model model.tflite --scorer huge-vocabulary.scorer -audio test.wav `

and received:

```
TensorFlow: v2.8.0-8-g06c8fea58fd
 Coqui STT: v1.4.0-alpha.1-0-gf3cdbd77
INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
Error on stat: 2
Unexpected type for udio: 0
```

Having never used STT or TensorFlow, I have no idea where to begin.  Any pointers?

**Comment by wasertech** [comment]
@ankushnarula Can you show us the output of `soxi` for `test.wav`? It seem you have an unexpected type of audio.
It should look something like this:
```bash
❯ soxi ~/.assistant/data/fr/SR/savewav_2022-04-11_17-18-22_828005.wav

Input File     : '/home/waser/.assistant/data/fr/SR/savewav_2022-04-11_17-18-22_828005.wav'
Channels       : 1
Sample Rate    : 16000
Precision      : 16-bit
Duration       : 00:00:07.68 = 122880 samples ~ 576 CDDA sectors
File Size      : 246k
Bit Rate       : 256k
Sample Encoding: 16-bit Signed Integer PCM
```
Make sure your audio validates the following requirements:
1. `Channels       : 1`
2. `Sample Rate    : 16000`
3. `Precision      : 16-bit`
4. `Bit Rate       : 256k`
5. `Sample Encoding: 16-bit Signed Integer PCM`
